Makefile

```diff
@@ -74,6 +76,8 @@ NPMFILES = \
 # Build defaults.
 GOOS ?= linux
 GOARCH ?= amd64
+GOOSES ?= "linux windows"
+GOARCHES ?= "amd64 arm64"
```

Comment:
can you add comments by both of these that say basically >to override at the cli do `GOOSES="\"darwin bsd\""`
I think the need to double/escape quote will otherwise be confusing

Reply:
Thank you, adding these comments too